codecparsers/nalReader.h

```diff
@@ -28,7 +28,9 @@ class NalReader : public BitReader
     NalReader(const uint8_t *data, uint32_t size);
 
     /*parse Exp-Golomb coding*/
+    bool readUe(uint32_t& v);
```

Comment:
Do we need to move it as private? Seems we just use uint32_t readUe() as public interface. Same as  bool readSe(int32_t& v) and bool read(uint32_t& v, uint32_t nbits) in bitreader.